weblab/config/settings/base.py

```diff
@@ -53,6 +53,7 @@
 LOCAL_APPS = [
     'accounts.apps.AccountsConfig',
     'entities.apps.EntitiesConfig',
+    'experiments',
```

Comment:
Because it doesn't have a custom app config set up... I learned a neater way to set these up though (in the app's `__init__.py`) so I'll change it to avoid confusion.